h.helm.md

```diff
@@ -4,122 +4,145 @@
 
 ## Helm in K8s
 
-### Creating a basic Helm chart
+### Creating a Basic Helm chart
 
 <details><summary>show</summary>
 <p>
 
 ```bash
-helm create chart-test ## this would create a helm 
+helm create chart-test ## this creates a basic helm chart
 ```
 
 </p>
 </details>
 
-### Running a Helm chart
-
+### Add the Bitnami repo at https://charts.bitnami.com/bitnami to Helm
```

Comment:
Se don't _need_ the bitnami repo, but it's a good exercise to know how to add a repository